squad/frontend/templates/squad/build.html

```diff
@@ -13,51 +13,72 @@ <h2>Metadata</h2>
+            </div>
+        </div>
+        <div id="suite-{{suite.id}}" class="panel-collapse collapse">
+            <div class="panel-body">
+                {% for env, statuses in details.items %}
+                {% for status in statuses %}
+                <div class="panel-group" id="accordion-{{env.id}}">
+                    <div class="panel">
+                        <a data-toggle="collapse" data-parent="#accordion-{{env.id}}" data-target="#{{status.id}}">
+                            <i class='fa fa-microchip'></i>
+                            {{env}}&raquo;
+                            <i class='{% if status.tests_fail > 0 %}fa fa-times text-danger {% else %}
+                            fa fa-check text-success {% endif %}'></i>
+                        </a>
+                        <div id="{{status.id}}" class="panel-collapse collapse">
+                            <div class="panel-body">
+                                <a href="{% testrun_suite_tests_url status %}">
+                                    <div class='col-lg-10 col-sm-10' title='Test results'>
+                                        <i class='fa fa-check-square-o'></i>
+                                        <span class="badge" data-toggle="tooltip" data-placement="top" title="Total">{{status.tests_total}} tests</span>
+                                        {% if status.tests_pass > 0 %}
+                                        <span class="badge alert-success" data-toggle="tooltip" data-placement="top" title="Pass">{{status.tests_pass}} pass</span>
+                                        {% endif %}
+                                        {% if status.tests_skip > 0 %}
+                                        <span class="badge alert-warning" data-toggle="tooltip" data-placement="top" title="Skip">{{status.tests_skip}} skip</span>
+                                        {% endif %}
+                                        {% if status.tests_fail > 0 %}
+                                        <span class="badge alert-danger" data-toggle="tooltip" data-placement="top" title="Fail">{{status.tests_fail}} fail</span>
+                                        {% endif %}
+                                    </div>
+                                </a>
+                                <a href="{% project_url status.test_run %}">
+                                    <div class='col-md-2 col-sm-2' title='Test run'>
+                                        <i class='fa fa-cog'></i>
+                                        {{status.test_run.job_id}}
+                                    </div>
+                                </a>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                {% endfor %}
+                {% endfor %}
+            </div>
+        </div>
     </div>
-</div>
-</a>
-{% endif %}
-{% endfor %}
+    {% endfor %}
 </div>
 {% if count_statuses_nofail > 0 %}
```

Comment:
this section is always empty in the UI.

Reply:
@mwasilew in fact i think this whole section needs to go now, since all results (passes+fails) are grouped by suite & env, hence they end up nested. As opposed to the previous state where all hierarchy is flattened, hence this previously made sense, while now no.
Also this is substituted for with the red x or the green check next to the suite and env headers inside a given accordion. eg. if on kselftest --> x86 grouping, one test run had fails while all others passed, we will end up with on red X among a number of green checks. That X will bubble up and be shown on both parent groupings: the env and its parent suite.